tests/perpetual/test_onboarding_payload.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+from eth_account import Account
+
+
+def test_onboarding_object_generation():
+    # all known values from authentication service tests
+    from x10.perpetual.user_client.onboarding import get_onboarding_payload
+
+    known_private_key = "50c8e358cc974aaaa6e460641e53f78bdc550fd372984aa78ef8fd27c751e6f4"
+    known_l2_public_key = "0x78298687996aff29a0bbcb994e1305db082d084f85ec38bb78c41e6787740ec"
+
+    l1_account = Account.from_key(known_private_key)
+
+    payload = get_onboarding_payload(
+        l1_account,
+        time=datetime.datetime(
+            year=2024,
+            month=7,
+            day=30,
+            hour=16,
+            minute=1,
+            second=2,
+            tzinfo=datetime.timezone.utc,
+        ),
+    ).to_json()
+
+    assert (
```

Comment:
[nit] It might be better to do it this way:
```
from hamcrest import assert_that, equal_to

assert_that(
        payload,
        equal_to(
            {
                # ....
            }
        ),
    )
```